projects/tensorflow/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER mihaimaruseac@google.com
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        curl \
+        python-dev \
+        rsync \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install Bazel
+RUN echo "deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
+RUN curl https://bazel.build/bazel-release.pub.gpg | apt-key add -
+RUN apt-get update && apt-get install -y bazel
+
+RUN git clone --depth 1 https://github.com/tensorflow/tensorflow tensorflow
+WORKDIR $SRC/tensorflow
+COPY build.sh $SRC/
+
+# Custom environment variables
+ENV TF_CPP_MIN_LOG_LEVEL=100
```

Comment:
Is this a runtime env variable? In that case it won't work, as this Dockerimage is used only for building the fuzzers, not for running. See https://github.com/google/oss-fuzz/blob/master/docs/fuzzer_environment.md

Reply:
Looks like there is no way to disable logging otherwise? In that case I'd suggest calling `setenv` from the fuzz target initialization code. Add something like the following to https://github.com/tensorflow/tensorflow/blob/6d9b2eb9c9e8607c43ba1f7c06d95e0b574986b3/tensorflow/core/kernels/fuzzing/fuzz_session.h#L26

```cpp
  extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) { \
    static int suppress_logging = setenv(...); \
    ...
```

As per libFuzzer's documentation: http://llvm.org/docs/LibFuzzer.html#startup-initialization

Reply:
> Hmm, when tested locally it seemed to also work during running.

I suspect because you used `infra/helper.py shell` to test this, the `shell` command loads the builder image and is supposed to be used for debugging build issues. There is `run_fuzzer` command which more or less emulates the real running environment. I'll upload a quick CL to clarify that in the help message and/or the docs.

Reply:
I wanted to not modify the code if possible, to not impede on the internal fuzzing we're also doing. But I guess I'll have to do that then

Reply:
Internal fuzzing will also benefit from this change :) It's a bit unfortunate that env variable is the only way to control logging, but changing that would require much bigger effort, I guess.